spec/spec_helper.rb

```diff
@@ -1,7 +1,6 @@
 require "bundler/setup"
 
 require "spyke"
-require "multi_json"
```

Comment:
again, not sure whether I should explicitly put `require "json"` here. Presumably if we're requiring it explicitly above, it's not needed here, right?